src/app/es6-polyfills.js

```diff
@@ -1,3 +1,5 @@
+// Reflect is needed to make Custom Element ES6 Classes working
```

Comment:
all browser apart of IE11 i think have it nativly so i think also this is just for IE

Reply:
oh okay, will add it